samples/WOCCatalog/WOCCatalog/PickersViewController.m

```diff
@@ -129,20 +95,19 @@ - (void)action:(id)sender
     }
 }
 
-- (CGRect)calcFrameForPicker:(CGSize)size
+- (CGRect)calcFrameForPicker
 {
     CGRect screenRect = [[UIScreen mainScreen] applicationFrame];
```

Comment:
This should probably be self.view.bounds -- the view should scale first, and its contents should not be based on screen size